auth_backend/http.py

```diff
@@ -0,0 +1,7 @@
+HTTP_200_OK = 200
+HTTP_201_CREATED = 201
+HTTP_204_NO_CONTENT = 204
+HTTP_404_NOT_FOUND = 404
+HTTP_409_CONFLICT = 409
+HTTP_422_UNPROCESSABLE_ENTITY = 422
+HTTP_500_INTERNAL_SERVER_ERROR = 500
```

Comment:
There's no need to do this as the requests library already has a long list of codes
eg.

`requests.codes.not_found`